src/Lab/MonitorPlayground/Program.cs

```diff
@@ -75,7 +76,7 @@ static void Main(string[] args)
                     window.Close();
                 }
 
-                input.Mice[0].Position = new PointF(window.Size.Width / 2f, window.Size.Height / 2f);
+                input.Mice[0].Position = new Vector2(window.Size.X / 2f, window.Size.Y / 2f);
```

Comment:
difficult to track which type is used where, but I believe there should be a `HalfSize`?

Reply:
Will take a look.

Reply:
Actually nah Size is a Vector2. However, I have been thinking of adding "aliases" to some vector properties but that's a discussion for another day.